includes/class-api.php

```diff
@@ -349,6 +349,9 @@ public function attach_featured_image( $post_id, $post_args ) {
 			return false;
 		}
 
+		// Allow download_url() to use an external host to retrieve featured image.
+		add_filter( 'http_request_host_is_external', array( $this, 'allow_sync_external_host' ), 10, 3 );
```

Comment:
Hey @benjaminmueller could you just let me know what this is for and how it works?